src/components/shared/ColonyAvatar/ColonyAvatar.tsx

```diff
@@ -22,25 +22,20 @@ const displayName = 'ColonyAvatar';
 
 const ColonyAvatar = ({
   colonyAddress,
-  avatarURL,
```

Comment:
Removing this prop since a) it's not used and b) doesn't make sense if the purpose of this wrapper component is to use the avatar from the colony object.